diff --git a/Install b/Install
index 9235766..f370d63 100755
--- a/Install
+++ b/Install
@@ -58,7 +58,13 @@ fi
 # read variables from the config file
 
 exec 4<&0
-/usr/local/mgi/live/lib/python/Configuration.py $CONFIGURATION tab > .Configuration.tab
+if [ -z "$LIB_PY_MISC" ]
+then
+	LIB_PY_MISC=/usr/local/mgi/live/lib/python
+	export LIB_PY_MISC
+fi
+
+$LIB_PY_MISC/Configuration.py $CONFIGURATION tab > .Configuration.tab
 exec < .Configuration.tab
 while read name value junk
 do
@@ -91,7 +97,7 @@ if [ ! -h bin/python ]; then
 	cd bin
 	echo "Linking to python in `pwd`"
 	ln -s $PYTHON python
-	cd $TOP
+	cd "$TOP"
 fi
 
 cd www
@@ -100,7 +106,7 @@ if [ -h Configuration ]; then
 		rm Configuration
 fi
 ln -s ../Configuration Configuration
-cd $TOP
+cd "$TOP"
 
 # compile python libraries and make sure they're readable
 
@@ -110,7 +116,7 @@ rm *pyc
 $PYTHON -c 'import compileall; compileall.compile_dir(".")'
 chgrp www *pyc
 chmod 750 *pyc
-cd $TOP
+cd "$TOP"
 
 # add links to python and the config file from selected directories
 
@@ -121,9 +127,9 @@ do
 	if [ -h python ]; then
 		rm python
 	fi
-	ln -s $PYTHON_REL python
+	ln -s "$PYTHON_REL" python
    
-	cd $TOP
+	cd "$TOP"
 done
 
 # support new-style link to homology.shtml minihome page, by temporarily
@@ -133,13 +139,13 @@ if [ -h homology.shtml ]; then
 	rm homology.shtml
 fi
 ln -s orthology.shtml homology.shtml
-cd $TOP
+cd "$TOP"
 
 # process the eGUL links.dat file
 
 cd www/egul/
 ./install $MGIHOME_URL
-cd $TOP
+cd "$TOP"
 
 # make the www/include directory and generate any include files
 
@@ -148,13 +154,13 @@ if [ ! -d include ]; then
 	echo "Making 'include' directory in `pwd`"
 	mkdir include
 fi
-cd $TOP/admin
-gen_rcd
-gen_includes
+cd "$TOP/admin"
+./gen_rcd
+./gen_includes
 #gen_includes.py
 
-cd $TOP
+cd "$TOP"
 
 # link to the stylesheet downloaded by the WI for its userdocs
 # (temporarily disabled until needed)
@@ -183,7 +189,7 @@ find www -type d -print | xargs chgrp www
 
 cd www
 includeLinker
-cd $TOP
+cd "$TOP"
 
 #
 # Warranty Disclaimer and Copyright Notice
diff --git a/admin/gen_includes b/admin/gen_includes
index a02e52a..04544e1 100755
--- a/admin/gen_includes
+++ b/admin/gen_includes
@@ -1,11 +1,14 @@
 #!./python
 import sys
+import os
+
 if '.' not in sys.path:
 	sys.path.insert (0, '.')
 
-MGI_LIBS = '/usr/local/mgi/live/lib/python'
-if MGI_LIBS not in sys.path:
-	sys.path.insert (0, MGI_LIBS)
+if "LIB_PY_MISC" not in os.environ:
+   sys.path.insert (0, '/usr/local/mgi/live/lib/python/')
+else:
+   sys.path.insert (0, os.environ["LIB_PY_MISC"])
 
 import Configuration
 config = Configuration.get_Configuration ('Configuration', 1)
@@ -13,7 +16,6 @@ config = Configuration.get_Configuration ('Configuration', 1)
 import homelib
 import webshare_lib
 
-import os
 import pg_db
 db = pg_db		# setup alias to minimize code changes
 import string
diff --git a/admin/gen_rcd b/admin/gen_rcd
index f1326cd..6004367 100755
--- a/admin/gen_rcd
+++ b/admin/gen_rcd
@@ -1,12 +1,15 @@
 #!./python
 
 import sys
+import os
+
 if '.' not in sys.path:
 	sys.path.insert (0, '.')
 
-MGI_LIBS = '/usr/local/mgi/live/lib/python'
-if MGI_LIBS not in sys.path:
-	sys.path.insert (0, MGI_LIBS)
+if "LIB_PY_MISC" not in os.environ:
+   sys.path.insert (0, '/usr/local/mgi/live/lib/python/')
+else:
+   sys.path.insert (0, os.environ["LIB_PY_MISC"])
 
 import webshare_util
 
