<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="utf-8" />
	<title>Tour of HMDC</title>
	<meta name="generator" content="BBEdit 10.5" />
</head>
<body>
<h1>A brief tour of the Human-Mouse: Disease Connection</h1>

<font size="3" color="red">Placeholder document; work in progress -Joanne</font>

<p>
<div id=TOC>
<h2>Table of contents</h2>
<br>
<ol>
	<li>Topic1</li>
	<li>Topic2</li>
</ol>
</div>
<br>
<div id=overview>
<h2>The Human-Mouse: Disease Connection</h2>

<p>The goal of the Human-Mouse: Disease Connection (HMDC) is to provide seamless human-to-mouse 
data traversal, enabling clinical and translational researchers to take advantage of wealth 
of data and annotations from mouse models; as well as allowing mouse researchers to connect 
their findings directly to genetic associations reported in human disease.</p>
<p>The mouse is genetically and physiologically similar to humans, is tractable as a 
laboratory animal, has a fully sequenced and well-annotated genome, and a readily available 
set of powerful molecular technologies for manipulating its genome in very precise ways. 
Now, clinical researchers whose primary focus is on human genetic disease, variants, and 
natural mutations have a highly-accessible way to explore experimentally characterized mouse 
mutants for a spectrum of associated phenotypes, as well as known disease models developed 
by the greater research community. Investigators can begin with genes (symbols, names or IDs) 
or gene lists, genome positions (as coordinates from human or mouse, or as <a href="http://en.wikipedia.org/wiki/Variant_Call_Format">.vcf</a>
 files), <a href="http://www.ncbi.nlm.nih.gov/omim">OMIM diseases</a> 
or phenotypes, and retrieve a list of genes, annotated mammalian phenotypes, associated human 
diseases and/or available mouse models for further research, along with comprehensive 
supporting references. If you are new to working with mouse models, or encounter terms 
which are unfamiliar to you, we also encourage you to browse the <a href="homepage_IntroMouse.shtml">
Introduction to Mouse Genetics</a> in order to aid with the interpretation of your results.</p>
<p>
<h3>What can you do with the HMDC?</h3>
<p>
This tool can be used to:
<ul>
	<li>use information about known mouse phenotypes to prioritize candidate genes from a 
	list generated by whole genome or exome sequencing</li>
	<li>use information about known mouse phenotypes to prioritize candidates within a 
	region (QTLs, GWAS, association or family based studies)</li>
	<li>identify mouse genes which exhibit a phenotype of interest</li>
	<li>identify human and mouse genetic associations to disease</li>
	<li>find an experimental mouse model to use in the study of a specific human disease</li>
</ul>

<h3>Beginning your search</h3>

<p>Entry into the HMDC is designed to be straightforward. On the homepage, three boxes appear 
where investigators may choose to enter (1) genes - either individually or multiples, (2) 
a genome location or set of regions, and/or (3) disease name(s) or mammalian phenotype(s). 

<div class="clear">
<img src="images/hmdc_search.png" width="600" height="245" alt="search boxes" border="1">
</div>
<p>
To enter multiple genes or genomic regions, use commas, spaces or new lines to separate entries. 
Human and mouse data may both be used, though for positions, you must specify a 
single organism. Please also ensure that you are using the current genome assembly (build) 
or the results will be inaccurate. If you need to convert your data, a <a href="http://www.ncbi.nlm.nih.gov/genome/tools/remap#tab=asm">simple online remapping tool 
has been provided by NCBI</a>. Gene names and symbols will automatically match official symbols, 
full names and synonyms 
in both species. For multiple disease or phenotype terms, use the autocomplete options for the 
greatest specificity. If you do not select from the autocomplete list, the search will run 
based on text-matching. Upload of variant call format (.vcf) files is also supported.<p>
<p>For many experimental questions, a single search box will be sufficient, but if two boxes 
have search terms entered, the results will reflect cases matching on both categories (boolean 
AND). While this can be an efficient shortcut, if a very precise phenotype or disease annotation 
is used, some potentially relevant results may be omitted.</p>
<p>Whichever search box is used, the same set of results tables will appear. To follow along
with this example, enter <i>trp53 proc apc cdkn2a</i> into the <b>Genes</b> search box. 


<div class="clear">
<h3>Search Results</h3>
<p>Along the top, an orange banner remembers your original query and allows you to quickly 
modify your results by adding, removing or replacing search terms. A second banner below 
serves as a header for your results, and indicates which type of matching was run. The results 
themselves are arranged into three tabbed tables for: Traits & Models <i>(shown)</i>, 
Genes, and Diseases.</p>
<img src="images/hmdc_geneSearchResult.png" width="600" height="302" alt="starting results page" border="1">
</div>
<br>
<br>
<div id=HomologyPD>
<b>1. Gene homology x Phenotypes/Diseases tab</b>
	<ul>
		<img src="images/hmdc_tab_grid.png" width="550" height="236" border="1">
		<li>A list of genes are returned as the rows in a grid. Human and mouse orthologs 
		are matched and displayed in adjacent columns of the same row. If two genes within a species 
		are members of the same <a href="http://www.informatics.jax.org/orthology.shtml">Vertebrate 
		Homology Class</a> (according to HomoloGene ID), both will be displayed in the same 
		row. If a homolog does not exist in one of the species, or for <a href="http://www.informatics.jax.org/glossary/transgene">transgenes</a>
		 where an artificial construct has been introduced (even if human or mouse genes are being expressed), a 
		blank cell will appear in the appropriate column. In each case, the gene name, 
			gene symbol or a published synonym of the gene name or symbol matched some part of the search input. Human orthologs 
			are listed in the first column, with mouse orthologs in the second. Note that human 
			genes are in all capital letters (i.e. APC), while murine nomenclature is in sentence case (i.e. Apc).</li>
		<li><font color="#FFA62F"><b><i>Please note:</i></b> If there are no phenotypes or disease associations reported for human or mouse homologs of a gene, 
			the row will not be displayed on the grid even if the text has matched. Use the 
			genes tab to retrieve a complete list.</font>
		<li>Systems level mammalian phenotypes (mouse) are shown at the to the left of a solid 
			grey bar, while associated human diseases or published disease models that involved
			mutant alleles of a given gene are shown to the right. Orange filled boxes indicating 
			human annotations will only appear in the associated human diseases and mouse 
			models section. Mammalian phenotypes use a hierarchical structured vocabulary, 
			search the <a href="http://www.informatics.jax.org/searches/MP_form.shtml">MP browser</a> 
			to find specific terms.</li>
		<li><b><i>Left side of grid:</i></b> If a mutant allele of a given gene has an annotation to an affected system,
			the box at the intersection of the gene and system will be colored in blue. Dark 
			shades indicate many annotations in that category, while lighter shades are fewer. 
			White cells indicate that no phenotypes or observations have been reported for that 
			gene in that system. Systems only appear if at least one gene (genotype) has an 
			annotation. <b>N</b> is used to indicate that the system was examined and 
			found to be normal. If researchers determined that the mouse carrying a particular 
			mutation was overall &lsquo;normal&rsquo;, a <i>normal phenotype</i> column will 
			be displayed on the far right of the Mammalian Phenotypes section. Keep in mind 
			that not all mutations disrupt gene function and as this table compresses multiple
			alleles into a single row, both normal and highly affected systems 
			may appear for the same gene.</li>
		<li><b><i>Right side of grid:</i></b> If mutations in this gene have been reported in human 
			patients (and that data has been collected by OMIM), an orange fill will be used 
			at the intersection of a gene with its associated human disease. If mutant mice 
			have been reported as disease models, a blue fill will be used. In cases where 
			both mouse models and human clinical cases have been used to support involvement 
			of orthologous genes, a two-toned fill will appear. White indicates that the intersection 
			of a given gene and human disease has not been reported. 
		<li>Unexpected diseases may appear in the grid if a phenotype or disease term was 
			used. For example, a search for <i>Alzheimer</i> will also return <i>Breast 
			Cancer</i> and <i>Schizophrenia</i>. This is because all of the diseases that 
			have been annotated to the allele pair which matches models for "Alzheimer Disease" 
			will be displayed, and in some cases like this one, <a href="http://www.informatics.jax.org/allele/MGI:2180364">
			Cav1<sup>tm1Mls/tm1Mls</sup></a> and <a href="http://www.informatics.jax.org/allele/MGI:2652485">
			Plcb1<sup>tm1Hssh/tm1Hssh</sup></a> have been reported to exhibit the 
			characteristic phenotypes of both diseases. Use the filters if these are not of 
			interest, or go to the <b>Diseases</b> tab to see only those diseases which 
			matched the original search terms with their gene annotations.</li>
		<li><b><i>Filtering:</i></b> Results can be filtered by clicking the boxes that appear 
			in each row and column, adding check marks. If a system, disease or gene 
			symbol match is not relevant to your experimental question, leave them unselected 
			and then click the <b>&#91;Submit Filters&#93;</b> button in the top-left. This will filter
			all tabs. Filters can be removed by clicking on the <b>&#91;(x) remove filters&#93;</b> 
			note which will appear above the tabs and just below the orange <b>Results</b> 
			header bar.</li>
		<li>Hover over squares to see the Gene by System intersection, and click individual 
			squares to generate a pop-up with allele pairs used to report this annotation 
			as well as more detailed phenotype annotations for that subset. Clicking on an 
			allele symbol or row will generate a new tab with all phenotype annotations and 
			indicate J:#s as a supporting reference. If multiple genotypes have reported 
			phenotypes (for example, a defined allele pair on different inbred backgrounds), 
			these will be concatenated. Click the allele symbol in the header to view this 
			as an <i>Allele Detail Page</i>. For help interpreting Allele Detail pages see 
			<a href="http://firien.informatics.jax.org/userhelp/ALLELE_detail_help.shtml">
			here</a>.<br>
			<img src="images/hmdc_popupDetail.png" width="300" height="163" border="1">
			<img src="images/hmdc_popupfullPheno.png" width="253" height="163" border="1"></li>
	</ul>
</div>	
<br>
<div id=GenesTab>
<b>2. Genes tab</b>
	<ul>
		<img src="images/hmdc_tab_genes.png" width="550" height="278" border="1">
		<li><font color="#FFA62F"><b><i>Please note:</b></i> The genes tab returns a list of all genes where the 
		gene terms searched match part or all of the gene symbol, gene name or synonyms of a 
		human or mouse gene. For phenotype and disease term searches, at least one genotype 
		which involves this gene has been annotated.</font></li>
		<li>To filter this list, apply filters on the "Gene Homology x Phenotypes/Diseases" 
		tab.</li>
		<li>On this tab, human and mouse genes are separated into 
		different rows, use the arrows in the headers for "Organism" (column 1) and/or "Marker 
		Symbol" (column 2) to sort. Click on the Gene Symbol in the second column to go 
		to a mouse <i>Gene Detail Page</i> on MGI in rows where <b>Organism: mouse</b> or 
		<i>Vertebrate Homology Page</i> in rows where <b>Organism: human</b>. Example: 
		<a href="http://www.informatics.jax.org/marker/MGI:88039">mouse Apc</a>
		and <a href="http://www.informatics.jax.org/homology/30950">human APC</a>.</li>
		<li>Data on this tab can be Downloaded using the button at the top of the table. Columns are 
		tab delimited and multiple "Associated Human Diseases" or "Abnormal Mouse Phenotypes
		Reported in these Systems" terms are pipe separated (&#124;).</li>
		<li>The Associated Human Diseases tab in a row that corresponds to <b>Organism: 
		mouse</b> will list diseases where mutant mice have been reported to display 
		phenotypes and symptoms matching the human disease. This also corresponds to a blue 
		fill in the right half "Gene homology x Phenotypes/Diseases" grid on the previous tab.</li>
		<li>The Associated Human Diseases tab in a row that corresponds to <b>Organism: 
		human</b> indicates that mutations have been reported in clinical cases or variant 
		associations have been made in human populations. This corresponds to orange fill 
		in the right half of the grid.</li>
		<li><b>Abnormal Mouse Phenotypes Reported in these Systems</b> only appear in rows where 
		<b>Organism: mouse</b>. This will retain the complete list of affected systems for 
		the genes that comprise this table, even if mammalian phenotype filters have been 
		applied to the grid.</li>
		<li>The <b>References</b> column provides a list of all MGI-curated references for 
		a gene, as well as a sub-list of Disease Relevant publications, where a mouse was 
		specifically reported by the authors as a model for one of the diseases represented 
		in the Associated Human Diseases column. Reference pages contain the full citation, 
		abstract, curated data, and direct links to the paper itself. These references 
		will focus on mouse genetic models as MGI does not curate human-only data.</li>
		<li>The right-most column on the Genes tab contains links to the <a href="http://www.findmice.org/">
		International Mouse Strain Resource (IMSR)</a> which is a database indexing all of 
		the major public and commercial repositories that distribute mice to the research 
		community. The number indicates how many mouse strains carrying mutant alleles of 
		a given gene are available for purchase. This may link to multiple unique alleles 
		and allele types, so take careful note of which allele is present before placing an 
		order to ensure that the expected phenotypes will be found. Order forms for each 
		strain are linked in the <b>Repository</b> column in the IMSR. If the "Find Mice (IMSR)"
		column on the Genes tab is blank, or the repositories listed with the IMSR do not 
		distribute your preferred model, it may still be  may also be possible to obtain 
		mice by directly contacting the corresponding author of a publication which developed 
		the line. See the Original Reference at the bottom of MGI's Allele Detail pages.</li>
	</ul>
</div>
<div id=DiseaseTab>
<b>3. Diseases</b>
	<ul>
		<img src="images/hmdc_tabDiseases.png" width="550" height="248" alt="Diseases tab" border="1">
		<li>The <b>Diseases</b> tab will present all of the diseases that have been 
		associated to the list of human or mouse genes matching the original search terms. 
		This includes both those associated with clinical cases or associations in human 
		data (see "Associated Human Markers" column to see if a human gene has been annotated) 
		and mouse models that have been reported as a model of this disease (see "Associated 
		Mouse Markers" column to see mouse genes and or mouse strain carrying transgenes 
		which have been annotated). If a phenotype term was used This tab can also be filtered by </li>
</ul>		


</div>

<div id=search1>

<img src="images/hmdc_Home.png">

The goal is to pull together both human and mouse data into a single flexible tool which 
can be used to answer a variety of biological and especially disease-related questions 
<p>
This tool can be used to:
-	use information about known mouse phenotypes to prioritize candidate genes from a list (whole genome sequencing, exome sequencing)
-	use information about known mouse phenotypes to prioritize candidates within a region (QTLs, GWAS, association or family based studies)
-	identify mouse genes and genetic which have been annotated to a disease
-	identify mouse genes which are known to be involved within a specific biological system
-	find an experimental mouse model to use in the study of a specific human disease
<p>
</div>

</body>
</html>
