<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="utf-8" />
	<title>Tour of HMDC</title>
	<meta name="generator" content="BBEdit 10.5" />
</head>
<body>
<h1>A brief tour of the Human-Mouse: Disease Connection</h1>

<font size="3" color="red">Placeholder document; work in progress -Joanne</font>

<p>
<div id=TOC>
<h2>Table of contents</h2>
<br>
<ol>
	<li>Topic1</li>
	<li>Topic2</li>
</ol>
</div>
<br>
<div id=overview>
<h2>The Human-Mouse: Disease Connection</h2>

<p>The goal of the Human-Mouse: Disease Connection (HMDC) is to provide seamless human-to-mouse 
data traversal, enabling clinical and translational researchers to take advantage of wealth 
of data and annotations from mouse models; as well as allowing mouse researchers to connect 
their findings directly to genetic associations reported in human disease.</p>
<p>The mouse is genetically and physiologically similar to humans, is tractable as a 
laboratory animal, has a fully sequenced and well-annotated genome, and a readily available 
set of powerful molecular technologies for manipulating its genome in very precise ways. 
Now, clinical researchers whose primary focus is on human genetic disease, variants, and 
natural mutations have a highly-accessible way to explore experimentally characterized mouse 
mutants for a spectrum of associated phenotypes, as well as known disease models developed 
by the greater research community. Investigators can begin with genes (symbols, names or IDs) 
or gene lists (including <a href="http://en.wikipedia.org/wiki/Variant_Call_Format">.vcf</a>
 files), genome positions (in human or mouse), <a href="http://www.ncbi.nlm.nih.gov/omim">OMIM diseases</a> 
or phenotypes, and retrieve a genes, annotated mammalian phenotypes, associated human 
diseases and/or available mouse models for further research, along with comprehensive 
supporting references. If you are new to working with mouse models, or encounter terms 
which are unfamiliar to you, we also encourage you to browse the <a href="homepage_IntroMouse.shtml">
Introduction to Mouse Genetics</a> in order to aid with the interpretation of your results.</p>
<p>
<h3>What can you do with the HMDC?</h3>
<p>
This tool can be used to:
<ul>
	<li>use information about known mouse phenotypes to prioritize candidate genes from a 
	list generated by whole genome or exome sequencing</li>
	<li>use information about known mouse phenotypes to prioritize candidates within a 
	region (QTLs, GWAS, association or family based studies)</li>
	<li>identify mouse genes which exhibit a phenotype of interest</li>
	<li>identify human and mouse genetic associations to disease</li>
	<li>find an experimental mouse model to use in the study of a specific human disease</li>
</ul>

<h3>Overview</h3>

<p>Entry into the HMDC is designed to be straightforward. On the homepage, three boxes appear 
where investigators may choose to enter (1) genes - either individually or multiples, (2) 
a genome location or set of regions and/or (3) disease name(s) or mammalian phenotype(s). 

<div class="clear">
<img src="images/hmdc_search.png" width="534" height="219" alt="begin search">
</div>
<p>
To enter multiple genes or genomic regions, use a new line for each entry or separate with 
commas. Human and mouse data may both be used, though for positions, you must specify a 
single organism. Please also ensure that you are using the current genome assembly (build) 
or the results will be inaccurate. If you need to convert your data, a <a href="http://www.ncbi.nlm.nih.gov/genome/tools/remap#tab=asm">simple online remapping tool 
has been provided by NCBI</a>. Gene names and symbols will automatically match official symbols and synonyms 
in both species. For multiple disease or phenotype terms, use the autocomplete options for the 
greatest specificity. If you do not select from the autocomplete list, the search will run 
based on text-matching. Upload of variant call format (.vcf) files is also supported.<p>
<p>For many experimental questions, a single search box will be sufficient, but if two boxes 
have search terms entered, the results will reflect cases matching on both categories (boolean 
AND). While this can be an efficient shortcut, if a very precise phenotype or disease annotation 
is used, some potentially relevant results may be omitted.</p>
<p>Whichever search box is used, the same set of results tables will appear. To follow along
with this example, enter <i>trp53 proc apc cdkn2a</i> into the <b>Genes</b> search box. 

<div class="clear">
<img src="images/hmdc_geneSearchResult.png" width="534" height="265">
</div>
<p>Along the top, an orange banner remembers your original query and allows you to quickly 
modify your results by adding, removing or replacing search terms. A second banner below 
serves as a header for your results, and indicates which type of matching was run. The results 
themselves are arranged into three tabbed tables for: Traits & Models <i>(shown)</i>, 
Genes, and Diseases.</p>
<ol>
	<li><b>Traits & Models tab</b>
		<ul>
			<li>A list of nine genes and one transgene are returned as the 
			rows in a grid. In each case, the gene name, gene symbol or a published 
			synonym of the gene name or symbol matched some part of the search input. Human orthologs 
			are listed in the first column, with mouse orthologs in the second. Note that human 
			genes are in all capital letters, while murine nomenclature is in sentence case.</li>
			<li>Systems level mammalian phenotypes (mouse) are shown at the to the left of a solid 
			grey bar, while associated human diseases or published disease models that involved
			mutant alleles of a given gene are shown to the right. Orange filled boxes will 
			only appear in the annotated human diseases and mouse models section.</li>
			<li><b><i>Left side of grid:</i></b> If a mutant allele of a given gene has an annotation to an affected system,
			the box at the intersection of the gene and system will be colored in blue. Dark 
			shades indicate many annotations in that category, while lighter shades are fewer. 
			White cells indicate that no phenotypes or observations have been reported for that 
			gene in that system. Systems only appear if at least one gene (genotype) has an 
			annotation. <b>N</b> is used to indicate that the system was examined and 
			found to be normal. In some cases, <b>N!</b> will be used to indicate background 
			sensitivity. If researchers determined that the mouse carrying a particular 
			mutation was overall &lsquo;normal&rsquo;, a <i>normal phenotype</i> column will 
			be displayed on the far right of the Mammalian Phenotypes section. Keep in mind 
			that not all mutations disrupt gene function and as this table compresses multiple
			alleles into a single row, both normal and highly affected systems 
			may appear for the same gene.</li>
			<li><b><i>Right side of grid:</i></b> If mutations in this gene have been reported in human 
			patients (and that data has been collected by OMIM), an orange fill will be used 
			at the intersection of a gene with its associated human disease. If mutant mice 
			have been reported as disease models, a blue fill will be used. In cases where 
			both mouse models and human clinical cases have been used to support involvement 
			of orthologous genes, a two-toned fill will appear. White indicates that the intersection 
			of a given gene and human disease has not been reported.</li>
			<li><b><i>Filtering:</i></b> Results can be filtered by clicking the boxes that appear 
			in each row and column, adding check marks. If a system, disease or gene 
			symbol match is not relevant to your experimental question, leave them unselected 
			and then click the <b>&#91;Submit Filters&#93;</b> button in the top-left. This will filter
			all tabs. Filters can be removed by clicking on the <b>&#91;(x) remove filters&#93;</b> 
			note which will appear just below the orange <b>Results</b> header bar.<p></li>
		</ul>
	</li>
	<li><b>Genes tab</b>
		<br>
	</li>
	<li><b>Diseases</b>
		<br>
	</li>
</ol>		


</div>

<div id=search1>

<img src="images/hmdc_Home.png">

The goal is to pull together both human and mouse data into a single flexible tool which 
can be used to answer a variety of biological and especially disease-related questions 
<p>
This tool can be used to:
-	use information about known mouse phenotypes to prioritize candidate genes from a list (whole genome sequencing, exome sequencing)
-	use information about known mouse phenotypes to prioritize candidates within a region (QTLs, GWAS, association or family based studies)
-	identify mouse genes and genetic which have been annotated to a disease
-	identify mouse genes which are known to be involved within a specific biological system
-	find an experimental mouse model to use in the study of a specific human disease
<p>
</div>

</body>
</html>
